lambda/models/state_machine/create_model.py

```diff
@@ -171,4 +189,35 @@ def handle_poll_create_stack(event: Dict[str, Any], context: Any) -> Dict[str, A
 def handle_add_model_to_litellm(event: Dict[str, Any], context: Any) -> Dict[str, Any]:
     """Add model to LiteLLM once it is created."""
     output_dict = deepcopy(event)
+    is_lisa_managed = event["create_infra"]
+
+    litellm_params = {
+        "api_key": "ignored"  # pragma: allowlist-secret not a real key, but needed for LiteLLM to be happy
+    }
+
+    if is_lisa_managed:
+        # get load balancer from cloudformation stack
+        litellm_params["model"] = f'openai/{event["modelName"]}'
+        litellm_params["api_base"] = f"http://{event['modelUrl']}/v1"  # model's OpenAI-compliant route
+    else:
+        litellm_params["model"] = event["modelName"]
+
+    litellm_response = litellm_client.add_model(
+        model_name=event["modelId"],
+        litellm_params=litellm_params,
+    )
+    litellm_id = litellm_response["model_info"]["id"]
+    output_dict["litellm_id"] = litellm_id
+
+    model_table.update_item(
+        Key={"model_id": event["modelId"]},
+        UpdateExpression="SET model_status = :ms, litellm_id = :lid, last_modified_date = :lm, model_url = :mu",
+        ExpressionAttributeValues={
+            ":ms": ModelStatus.IN_SERVICE,
+            ":lid": litellm_id,
+            ":lm": int(datetime.utcnow().timestamp()),
+            ":mu": litellm_params.get("api_base", ""),
```

Comment:
I think if I change this to None we can get consistent `null` in the UI if this is problematic

Reply:
Since it's already a string field, this didn't do anything- ddb will always respond with an empty string here.